rio_tiler/colormap.py

```diff
@@ -293,7 +290,7 @@ def get(self, name: str) -> Dict:
             colormap = numpy.load(cmap)
             assert colormap.shape == (256, 4)
             assert colormap.dtype == numpy.uint8
-            return {idx: value.tolist() for idx, value in enumerate(colormap)}
+            return {idx: tuple(value) for idx, value in enumerate(colormap)}  # type: ignore
```

Comment:
`value` entry in the numpy array could be of any length (which is why we add an `assert` test) but mypy is not happy ;-)